tests/Controller/CRUDControllerTest.php

```diff
@@ -3708,4 +3651,60 @@ private function expectTranslate(
             ->with($this->equalTo($id), $this->equalTo($parameters), $this->equalTo($domain), $this->equalTo($locale))
             ->willReturn($id);
     }
+
+    private function createTwig(): Environment
+    {
+        $templatingRenderReturnCallback = $this->returnCallback(function (
+            string $name,
+            array $context = []
+        ): string {
+            $this->template = $name;
+
+            $this->parameters = $context;
+
+            return '';
+        });
+
+        $twig = $this->getMockBuilder(Environment::class)
```

Comment:
Is this mock covering any expectation? If not, please use `createStub()` instead.